internal/slack/slack_client.go

```diff
@@ -0,0 +1,12 @@
+// Package slackbot provides a unified interface for Slack client operations
+package slackbot
+
+import "github.com/tuannvm/slack-mcp-client/internal/security"
+
+// SlackRunner defines the interface for a Slack client that can run and handle security
```

Comment:
I'm not so sure we need this interface. Security should be a feature we can toggle on and off, rather than a separate implementation for secured Slack.